servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/reporter/HttpReporter.java

```diff
@@ -127,7 +127,12 @@ private PublisherSource.Processor<Span, Span> initReporter(final Builder builder
         if (!builder.batchingEnabled) {
             buffer = newPublisherProcessorDropHeadOnOverflow(builder.maxConcurrentReports);
             spans = fromSource(buffer)
-                    .map(span -> allocator.wrap(spanEncoder.encodeList(Collections.singletonList(span))));
+                    .map(span -> {
+                        // Always encode spans as list: https://github.com/apple/servicetalk/pull/2092
+                        final byte[] bytes = spanEncoder.encodeList(Collections.singletonList(span));
+                        LOGGER.trace("Encoded received span: {}, result={} bytes", span, bytes.length);
```

Comment:
consider using the format `key='value'`, helps with parsing to key-value strings for various formatters.